yandexcloud/_sdk.py

```diff
@@ -22,8 +25,14 @@ def client(self, stub_ctor, interceptor=None):
             channel = grpc.intercept_channel(channel, self._default_interceptor)
         return stub_ctor(channel)
 
-    def waiter(self, operation_id):
-        return _operation_waiter.operation_waiter(self, operation_id, timeout=None)
+    def waiter(self, operation_id, timeout=None):
+        return _operation_waiter.operation_waiter(self, operation_id, timeout)
+
+    def wait_for_operation(self, operation_id, timeout=None, print_to_stream=None):
+        return _operation_waiter.wait_for_operation(self, operation_id, timeout, print_to_stream)
+
+    def get_operation_result(self, operation, response_type=None, meta_type=None, timeout=None, logger=None):
```

Comment:
It's not very obvious that this function will wait. Should we rename it somehow?

Reply:
Done